src/components/ChatEntry.js

```diff
@@ -1,21 +1,28 @@
 import React from 'react';
 import './ChatEntry.css';
+import TimeStamp from './TimeStamp';
 import PropTypes from 'prop-types';
 
 const ChatEntry = (props) => {
   return (
     <div className="chat-entry local">
-      <h2 className="entry-name">Replace with name of sender</h2>
+      <h2 className="entry-name">{props.sender}</h2>
       <section className="entry-bubble">
-        <p>Replace with body of ChatEntry</p>
-        <p className="entry-time">Replace with TimeStamp component</p>
-        <button className="like">🤍</button>
+        <p>{props.body}</p>
+        <p className="entry-time"> <TimeStamp time={props.timeStamp}></TimeStamp> </p>
+        <button className="like" onClick={() => props.onLikeMessage(props.id)}>🤍</button>
```

Comment:
Since you're not passing the `liked` prop into this component, you won't be able to set the red vs white heart emoji using that prop. If you did pass the prop, you could set that like 
```javascript
{props.liked ?  <red-heart-emoji-goes-here> : 🤍}
```